### PR TITLE
 update to latest babel 

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: config/webpack.config.js [hunk 1]
```diff
@@ -43,22 +43,12 @@ module.exports = {
 							// TODO: Look into specifying our minimum versions in preset-env
 							// for enhanced experience instead of making everything become ES5
 							presets: [
-								require.resolve('babel-preset-es3'),
-								require.resolve('babel-preset-env')
+								require.resolve('@babel/preset-env')
```
Comment (notlee) [inline config/webpack.config.js]: Do we need to specify [targets](https://babeljs.io/docs/en/babel-preset-env#targets) now?

File: test/integration/build/build.test.js [hunk 2]
```diff
@@ -285,52 +285,6 @@ describe('obt build', function () {
 			});
 		});
 
-		describe('component using bower dependency\'s js', function () {
```
Comment (notlee) [inline test/integration/build/build.test.js]: Is this test removed by mistake?

File: test/unit/tasks/demo-build.test.js [hunk 3]
```diff
@@ -151,9 +151,11 @@ describe('Demo task', function () {
 			fs.writeFileSync('demos/src/test2.mustache', '<div>test2</div>', 'utf8');
 			return demo()
 				.then(function () {
+					// console.log(fs.readFileSync('demos/local/demo.js', 'utf8'));
```
Comment (notlee) [inline test/unit/tasks/demo-build.test.js]: whoopsie

File: config/webpack.config.js (hunk 1 above)
Comment (JakeChampion) [reply config/webpack.config.js]: >if no targets are specified, @babel/preset-env will transform all ECMAScript 2015+ code by default.

I guess we could use a target of:
```{
  "targets": {
    "ie": "11"
  }
}
```

File: test/unit/tasks/demo-build.test.js (hunk 3 above)
Comment (JakeChampion) [inline test/unit/tasks/demo-build.test.js]:
```suggestion
```

File: test/unit/tasks/demo-build.test.js [hunk 4]
```diff
@@ -151,9 +151,11 @@ describe('Demo task', function () {
...
+					// process.exit(1);
```
Comment (JakeChampion) [inline test/unit/tasks/demo-build.test.js]:
```suggestion
```

File: config/webpack.config.js (hunk 1 above)
Comment (notlee) [reply config/webpack.config.js]: 👍

**Pull request merged**